src/dotnet/dotnet-fable/StateUtil.fs

```diff
@@ -227,6 +227,6 @@ let startAgent () = MailboxProcessor<Command>.Start(fun agent ->
         | None ->
             return! loop checker state
     }
-    let checker = FSharpChecker.Create(keepAssemblyContents=true, msbuildEnabled=false)
```

Comment:
So this is not needed now? I don't remember why we had to add it, probably for the .fsx scripts. Which I've disabled temporarily btw, as Fable uses now Dotnet.ProjInfo to crack the .fsproj files and I didn't have time to fix it for scripts yet (I think most people use .fsproj now anyways).